src/vm/exceptionhandling.h

```diff
@@ -389,15 +389,15 @@ class ExceptionTracker
 
     bool IsStackOverflowException();
 
-#ifdef FEATURE_PAL
+#if defined(FEATURE_PAL) && !defined(CROSS_COMPILE)
```

Comment:
Yes. I got build error

```
/nfs/rbp3/coreclr/src/vm/exceptionhandling.h:396:41: error: comparison of distinct pointer types ('CONTEXT *' (aka '_CONTEXT *') and 'PTR_CONTEXT' (aka '_T_CONTEXT *'))
        _ASSERTE(ex->GetContextRecord() == m_ptrs.ContextRecord);
                 ~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~~~
/nfs/rbp3/coreclr/src/inc/debugmacros.h:54:39: note: expanded from macro '_ASSERTE'
  #define _ASSERTE(expr) _ASSERTE_MSG(expr, #expr)
                                      ^
/nfs/rbp3/coreclr/src/inc/debugmacros.h:45:20: note: expanded from macro '_ASSERTE_MSG'
             if (!(expr)) {                                                 \
```

And I could get checked/debug build by excluding this method only. I think if we add excluded component in #8918, we can get more build error related to exception handling.